### DISCUSSION TITLE
header with text i don't want

### DISCUSSION THREAD

**Discussion by arkunis** [spatie/browsershot#825] [Q&A]
![image](https://github.com/spatie/browsershot/assets/126088452/283d1e53-b032-44bd-a079-0311a079908d)
Hi, i have the text on top of my pdf generated, how i can disable it ? 

i use symfony :

`        $pdf = Browsershot::html($this->render('wp_admin/pdf.html.twig', [
            'nom'=>$nom,
            'prenom'=>$prenom,
            'tel'=>$tel,
            'email'=>$email,
            'societe'=>$societe,
            'dates'=>$dates,
            'nbrParticipant'=>$nbrParticipant,
            'forfait'=>$forfait,
        ]), "devis.pdf")->pdf();

       $response = new Response($pdf);
        $response->headers->set('Content-Type', 'application/pdf');
        $response->headers->set('Content-Disposition', 'inline; filename="devis"');
        $response->headers->set('Content-Transfer-Encoding', 'binary');
        $response->headers->set('Cache-Control:', 'no-store, private');
        return $response;
`

thx

**Comment by thisiskode** [comment]
I've just got the same but it's not on all of them, despite the code being the same.

Did you resolve it?

**Reply by arkunis** [reply]
no i cheat i add white background on it ...

**Reply by thisiskode** [reply]
I see, good idea. At the moment I've had to revert back to wkhtmltopdf as I can't get browsershot working on the production server and don't have the time to investigate it.
